frontend/src/app/modules/home/organization/organization-menu/organization-menu.component.html

```diff
@@ -65,4 +65,4 @@
         </p-dropdown>
     </div>
 </div>
-<app-create-organization [(display)]="display" (organizationCreate)="organizationCreated($event)"></app-create-organization>
+<app-create-organization [(display)]="display" (organizationCreate)="organizationCreated($event)" [hasOrganization]="organizations.length !== 0"></app-create-organization>
```

Comment:
Could you use just organizations.length here?

Reply:
Fixed